### DISCUSSION TITLE
Climatization with ID.3

### DISCUSSION THREAD

**Discussion by Kaputt4** [tillsteinbach/VWsFriend#65] [Q&A] [answered]
Does climatization work with ID.3 Pro? I've been able to configure everything but it seems that the car is not providing any info about the climatization, although it's being used. The single row present in Postgres' _climatization_ table is from the moment I first runned VWsFriend and state is OFF.
Consequently, _Remaining Climatization Time_ shows always 0 mins.
It's also not working on _HomeKit_. All adapters and accessories are added properly but when I set any value on the climatization, the car doesn't do anything.
Anyone could help me or tell me what else could I check to search for the possible problem? Thanks!

**Comment by tillsteinbach** [comment]
Hmmm, this looks partly broken.
I just checked and what works is: Enabling through WeConnect ID App
What also works is disabling through HomeKit. So there is a problem with enabling it through HomeKit. I will investigate that as soon as possible.

**Reply by tillsteinbach** [reply]
I might have located the problem but I have a lot of work right now. It might need some days until I have released a fix. Sorry!

**Reply by Kaputt4** [reply]
Thanks for the help and checks!
Don't worry, take your time. You do such a great job, thanks a lot for sharing it! 😊

**Reply by tillsteinbach** [reply]
Thank you for your warm words! It looks like I found the bug. A fix is now in testing and I will release it with the next version.

**Comment by tillsteinbach** [comment, marked as answer]
This should be fixed in 0.6.0 release

**Reply by Kaputt4** [reply]
First checks and it is! I'll keep on testing, thank you so much.
